scripts/ci-build-azure-ccm.sh

```diff
@@ -28,6 +28,12 @@ source "${REPO_ROOT}/hack/ensure-go.sh"
 # shellcheck source=hack/parse-prow-creds.sh
 source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 
+: "${AZURE_STORAGE_ACCOUNT:?Environment variable empty or not defined.}"
+: "${AZURE_STORAGE_KEY:?Environment variable empty or not defined.}"
+: "${REGISTRY:?Environment variable empty or not defined.}"
```

Comment:
We repeat this same line below, at line 37.

Reply:
Thanks for reminding. Removed